### PR TITLE
Remove testcontainers, use docker-compose plugin

### PR DESCRIPTION
**Pull request by yevhenii0**
At the moment we have same job done by both testcontainers and docker-compose plugin, keep a single approach for integration tests. Also, since we dropped testcontainers, junit4 dep can be excluded as well, which means 1 `@Test` annotation in the classpath 😉  

Additionally removing xRequestId support which pollutes build logs with deprecation warnings.

### CODE REVIEW

File: integration-tests/build.gradle [hunk 1]
```diff
@@ -30,10 +34,34 @@ dependencies {
     testImplementation libraries.postgresql
     testImplementation libraries.mariadbJavaClient
     testImplementation libraries.springKafka
-    testImplementation libraries.testcontainers
     testImplementation libraries.awaitility
 }
 
+// junit4 deps are not used, exclude those
+configurations.testImplementation.exclude group: 'junit', module: 'junit'
```
Comment (onukristo) [inline integration-tests/build.gradle:41]: In some projects, I have used central registry for unwanted libs. For example in Webapp and in Payout: https://github.com/transferwise/payout-service/blob/master/build.common.gradle#L61

The benefit of that approach is, that it is in one place and prevents any kind of accidental transitive dependency coming in for any reason.

File: integration-tests/src/test/resources/docker-compose.yml [hunk 2]
```diff
@@ -52,3 +52,10 @@ services:
     command: "mysqld --ssl=0
     --character-set-server=utf8mb4
     --collation-server=utf8mb4_unicode_ci --transaction-isolation=READ-COMMITTED --innodb_autoinc_lock_mode=2"
+  postgres:
+    image: postgres:10.4
```
Comment (onukristo) [inline integration-tests/src/test/resources/docker-compose.yml:56]: Lets up to 12, we have only couple of database not upgraded to it.

**Review by onukristo — APPROVED** [review]

File: integration-tests/build.gradle (hunk 1 above)
Comment (yevhenii0) [reply integration-tests/build.gradle]: Thanks, moved to build.common.gradle

File: integration-tests/src/test/resources/docker-compose.yml (hunk 2 above)
Comment (yevhenii0) [reply integration-tests/src/test/resources/docker-compose.yml]: 👌

**Pull request merged**